src/Pages/Home/Home.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import Nav from '../../Components/Nav/Nav';
+import CategoryCard from '../../Components/CategoryCard/CategoryCard';
```

Comment:
Have a index.js file in components folder and import all your components there and from there export all the components, so this will keep your imports clean eg:
import {Nav, CategoryCard} from "../../components"

Reply:
Got it. Thanks, Rohan.